packages/leafygreen-provider/src/BaseFontSizeProvider.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { createContext, useContext } from 'react';
+import PropTypes from 'prop-types';
+
+interface BaseFontSizeProviderProps {
+  children: React.ReactNode;
+  baseFontSize: number;
+}
+
+const BaseFontSizeContext = createContext<number>(14);
+
+export function useBaseFontSize() {
+  return useContext(BaseFontSizeContext);
+}
+
+function BaseFontSizeProvider({
+  children,
+  baseFontSize,
+}: BaseFontSizeProviderProps) {
+  return (
+    <BaseFontSizeContext.Provider value={baseFontSize}>
+      {children}
+    </BaseFontSizeContext.Provider>
+  );
+}
+
+BaseFontSizeProvider.displayName = 'BaseFontSizeProvider';
+
+BaseFontSizeProvider.propTypes = {
+  children: PropTypes.node,
+  baseFontSize: PropTypes.number,
```

Comment:
Left a comment below in regards to your interface, but this should be only the numbers `14` and `16`

Reply:
👍